site/README.md

```diff
@@ -73,25 +73,19 @@ When running in development environments, avatars go to the `dev/avatars/(user.i
 
 ### Serving Blocks
 
-Before serving any blocks, they need to be built. Blocks can be registered in the repo's `/hub`
-with a build-config. The build-script `/site/scripts/build-blocks.sh` allows to build blocks
-individually. It requires the commandline tool:
-
-- `curl`
-- `jq`
-- `md5sha1sum`
-- `rsync`
-
-These can be installed by your cli pkg mngr of choice (use `brew` on macOS).
+Before serving any blocks, they need to be prepared (i.e. built in most cases).
+Blocks can be registered in the repo's `/hub` with a build-config.
+The build-script `yarn exe yarn exe site/scripts/prepare-blocks.ts` allows to prepare blocks.
```

Comment:
Might as well tidy this up while we're here
```suggestion
The build-script `yarn exe yarn exe site/scripts/prepare-blocks.ts` prepares blocks.
```

Reply:
👍 Addressed in https://github.com/blockprotocol/blockprotocol/pull/231/commits/257b032aace29c19ed0ac16e1e8683cc6a8809aa